src/bootstrap/bootstrap.py

```diff
@@ -171,7 +171,7 @@ def download_stage0(self):
             if os.path.exists(self.bin_root()):
                 shutil.rmtree(self.bin_root())
             filename = "rust-std-{}-{}.tar.gz".format(channel, self.build)
-            url = "https://static.rust-lang.org/dist/" + self.stage0_rustc_date()
+            url = "https://dev-static.rust-lang.org/dist/" + self.stage0_rustc_date()
```

Comment:
This is sort of up in the air as a convention. The old buildbot code would deploy to the archives and then we'd manually do the full release (change the manifests and put artifacts at `dist/*.tar.gz`. Now with rust-central-station *nothing* goes to prod on stable until we run the command manually. That means that until we actually do the release we wouldn't be able to create a new beta and/or nightly.

I'm personally thinking:

* We create a new stable build
* Stable uploads to dev
* Create a new beta bootstrapping from dev
* Beta is uploaded to prod
* Create a new nightly bootstrapping from beta (on prod)
* Do a release
* Send a PR to beta to bootstrap from prod

One extra PR but allows us to get beta/nightlies sooner, which I sorta like. Thoughts @brson?